.rancher-pipeline.yml

```diff
@@ -0,0 +1,28 @@
+stages:
+  - name: Build
+    steps:
+      - runScriptConfig:
+          image: maven:3-openjdk-8
+          shellScript: mvn clean install -DskipTests
+  - name: Build Docker
+    steps:
+      - publishImageConfig:
+          dockerfilePath: ./Dockerfile
+          buildContext: .
+          tag: docker.dev.folio.org/mod-circulation:core-functional-latest
```

Comment:
Given this is not intended to be used from the mainline, and this value should be specific to each team / group of work. Could this be left blank on the mainline in order to avoid potential collisions between teams?

Reply:
Ok, than this file does not make sense for me. Because default configuration won't allow me to deploy my feature without applying some changes to the file and than reverting them before merge. 

It adds an overhead rather brings a value, I'll remove this file.

Reply:
It was only a suggestion. I don't think you need to remove the file.

The default configuration only aids a single team. I don't really understand why this is desirable given the intent of the new development team environments. I'm ok with trying it given you or others find it useful.